src/shared/lib/tracking.ts

```diff
@@ -0,0 +1,48 @@
+import analytics from 'universal-ga';
+import $ from 'jquery';
+import * as _ from 'lodash';
+
+const trackingCode = 'UA-85438068-1';
+
+analytics.initialize(trackingCode,{ debug:true });
+
+const allowedHosts = ['www.cbioportal.org/beta'];
+
+const win = (window as any);
+
+function setCustomTrackingEvents(){
+
+    if (win.cancerStudyIdList !== 'null') {
```

Comment:
this is ugly, i know.  when we get to single page app, we will be able to know better where we are in site based on url.  router will be able to fire a lot of these events.